query-engine/js-connectors/js/neon-js-connector/src/neon.ts

```diff
@@ -9,7 +9,7 @@ neonConfig.webSocketConstructor = ws
 
 const debug = Debug('prisma:js-connector:neon')
 
-export type PrismaNeonConfig = ConnectorConfig & Partial<Omit<NeonConfig, 'connectionString'>>
+export type PrismaNeonConfig = ConnectorConfig & Partial<Omit<NeonConfig, 'connectionString'>> & { httpMode?: boolean }
```

Comment:
Nit: could we replace the `httpMode?: boolean` with a `mode?: 'http' | 'ws'`? This would be more explicit.